app/views/subscriptions/index.slim

```diff
@@ -0,0 +1,159 @@
+        h4.my-0.font-weight-normal
+          ' Basic
+      .card-body
+        h1.card-title.pricing-card-title Free
+        ul.list-unstyled.mt-3.mb-4
+          li Basic analytics
+          li Ad-supported
+      .card-footer
+        .d-flex
+          button.btn.btn-lg.btn-outline-light.disabled.mr-2.flex-fill.tip(
+            style='cursor: default'
+            title="Active since #{current_user&.created_at&.to_date}"
+          ) Active
+          button.btn.btn-lg.btn-outline-light.tip-top.disabled title='This plan lasts forever.' style='cursor: default'
+            = icon('fas', 'infinity')
+    .card
+      .card-header
+        h4.my-0.font-weight-normal
+          ' Gold
+      .card-body
+        h1.card-title.pricing-card-title
+          ' $4
+          small.text-muted
+            ' / mo
+        ul.list-unstyled.mt-3.mb-4
+          li Disable ads
+          li Predict when you'll PB
+      .card-footer
+        - if plan && plan.stripe_plan_id == ENV['STRIPE_PLAN_ID_TIER1']
+          - if plan.canceled?
+            button.btn.btn-lg.btn-block.btn-outline-light.disabled.tip(
+              title='Your subscription is canceled. It will last until the end of your billing period.'
+              style='cursor: default'
+            ) Canceled
+          - else
+            .d-flex
+              button.btn.btn-lg.btn-outline-light.disabled.flex-fill.mr-2.tip(
+                style='cursor: default'
+                title="Active since #{plan.created_at.to_date}"
+              ) Active
+              = form_for(:subscription, method: :delete) do |f|
+                = f.button(icon('fas', 'times'),
+                  data: {disable_with: icon('fas', 'circle-notch', class: 'fa-spin').html_safe}, \
+                  class: 'btn btn-lg btn-outline-danger tip', \
+                  title: 'Cancel your subscription. You will keep your benefits until we would have next billed you.' \
+                )
+        - elsif plan
+          .d-flex
+            button.btn.btn-lg.btn-outline-light.flex-fill.mr-2.disabled.tip(
+              style='cursor: default'
+              title='A strictly better subscription is making you inherit this one.'
+            ) Inherited
+            = form_for(:subscription, method: :patch) do |f|
+              = f.hidden_field(:stripe_plan_id, value: ENV['STRIPE_PLAN_ID_TIER1'])
+              = f.button(icon('fas', 'angle-double-down').html_safe, \
+                data: {disable_with: icon('fas', 'circle-notch', class: 'fa-spin').html_safe}, \
+                class: 'btn btn-lg btn-block btn-outline-warning tip', \
+                title: 'Downgrade your subscription to Gold level immediately. The prorated difference in price will be discounted from your next bill.' \
+              )
+        - else
+          / TODO: disable_with doesn't work in button_tag for some reason
+          = button_tag(current_user ? 'Subscribe' : 'Sign in to subscribe', disabled: current_user.nil?, data: { \
```

Comment:
I believe with the tag methods you can pass html attributes through the `html: {}` hash.

Reply:
My environment's a little messed up right now until this merge, so I'll check this out later since it's not vital to the branch getting deployed.